### ISSUE TITLE
[Bug] Window buttons are not displayed (Ubuntu 14.04 Gnome)

### ISSUE THREAD

**Issue by mradionov** [Soundnode/soundnode-app#372]
Actual:

![image](https://cloud.githubusercontent.com/assets/2007370/9915737/63f6f122-5cc1-11e5-8196-7628910ec5ab.png)

Expected:

![image](https://cloud.githubusercontent.com/assets/2007370/9915869/34a953fa-5cc2-11e5-981a-a0bc3999b7f0.png)

Version: Release 0.5.8 for Linux
Downloaded from: soundnodeapp.com
OS: Ubuntu 14.04 x64, Gnome Desktop

Console output:

```
$ ./Soundnode-App --debug
[9893:0916/222432:ERROR:browser_main_loop.cc(170)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
ATTENTION: default value of option force_s3tc_enable overridden by environment.
[9893:0916/222433:INFO:CONSOLE(86)] ""User is saved"", source: file:///tmp/.org.chromium.Chromium.FQx7I2/public/js/system/core.js (86)
```

**Comment by Pitros** [comment]
This is already fixed by #368 - it was reported 2 days ago #367.

@weblancaster keep this alive untill new release, so no more issues like that will be made.

**Comment by mradionov** [comment]
I can confirm it works for me with master build. Sorry for duplicate though, should have checked the master version first. Thanks.
